tests/schema/tests.py

```diff
@@ -1908,6 +1908,45 @@ def test_add_textfield_unhashable_default(self):
             editor.add_field(Author, new_field)
 
     @unittest.skipUnless(connection.vendor == 'postgresql', "PostgreSQL specific")
+    def test_add_field_with_index_with_charfield(self):
```

Comment:
`test_add_charfield_with_index` seems like a more appropriate name and follows the format used in previous test methods.